tests/test_config.py

```diff
@@ -17,19 +24,49 @@ def test_configure_logger_verbose():
     assert result == "Logger 'tests.test_config' configured with level=DEBUG"
 
 
-def test_configure_sentry_no_env_variable(monkeypatch):
-    monkeypatch.delenv("SENTRY_DSN", raising=False)
-    result = configure_sentry()
+def test_configure_sentry_env_variable_is_none():
```

Comment:
We should also include a test for the `sentry_dsn` value being the string `"None"`, since that's what it will be when read from the Dev1 env.